packages/dd-trace/src/plugins/database.js

```diff
@@ -4,6 +4,17 @@ const StoragePlugin = require('./storage')
 
 class DatabasePlugin extends StoragePlugin {
   static get operation () { return 'query' }
+
+  createSQLInjectionComment = () =>  {
```

Comment:
Again, let's try to avoid calling this "sql injection". Maybe `createDBMComment`?

Reply:
I think this method would look a little cleaner with this syntax:

```js
    createDBMComment() {
      //  body
    }
```

Reply:
updated this to
` createDBMPropagationComment() {
}`